1_fetch_spatial.R

```diff
@@ -0,0 +1,187 @@
+source("1_fetch/src/Download_nhd.R")
+source('1_fetch/src/download_states_shp.R')
+source('1_fetch/src/fetch_nhdplus_data.R')
+
+p1_sp_targets_list <- list(
+  
+  # Lake locations (lat/lon) Fetch #
+  
+  ## Lake sf dataset from sharepoint - stakeholder provided
```

Comment:
```suggestion
  ## Lake sf dataset from sharepoint - project team provided
```

Reply:
Let's avoid using "stakeholder" https://www2.gov.bc.ca/gov/content/governments/services-for-government/service-experience-digital-delivery/web-content-development-guides/web-style-guide/writing-guide/writing-guide-for-indigenous-content/terminology#word-choice